text/0001-script-based-tokenizer.md

```diff
@@ -0,0 +1,191 @@
+- Title: Script Based Tokenizer
+- Start Date: 2020-10-27
+- specification PR: meilisearch/specifications#2
+- Meilisearch Issue: meilisearch/Meilsearch#624
+
+## Feature Description and Interaction
+
+### Summary
+
+Tokenizer's principal role is to split documents into words (tokens) so each document can be indexed by it's contained words. It is also needed to split a search query into tokens, and search in the words index.
```

Comment:
About the first part of the sentence, is this a reference to reverse-indexing?

About the second part of the sentence, I do not understand it :s 
` It is also needed to split a search query into tokens, and search in the words index.`

This is my suggestion on the summary:
```suggestion
Tokenizer's principal role is to split documents into words (tokens). Each word becomes a key that has a list of all the documents in which it is present as its value. This is called [reverse-index](https://semify.com/?Reverse-Index&AID=1431#:~:text=A%20reverse%20index%20is%20a,inserting%20them%20into%20the%20index.).
The same tokenizer is used to split a search query into tokens. Those tokens will in turn be used by MeiliSearch to search for matching documents inside the reverse-index previously created.

Some languages do not split the same way as others, the aim of this tokenizer is to create a more diverse tokenizer. 
```